landsat/image.py

```diff
@@ -64,10 +68,10 @@ class BaseProcess(VerbosityMixin):
 
     """
 
-    def __init__(self, path, bands=None, dst_path=None, verbose=False, force_unzip=False, bounds=None):
+    def __init__(self, path, bands=None, out_crs=None, dst_path=None, verbose=False, force_unzip=False, bounds=None):
 
-        self.projection = {'init': 'epsg:3857'}
-        self.dst_crs = {'init': u'epsg:3857'}
+        self.projection = {'init': u'epsg:'+str(out_crs)} if isinstance(out_crs, object) else {'init': 'epsg:3857'}
```

Comment:
@KAPPS- these are incorrect. You are embedding a dict inside a dict. The error is not a landsat error, but a rasterio error because it receives incorrect projection and crs.